source/grid/grid_in.cc

```diff
@@ -844,11 +852,19 @@ GridIn<dim, spacedim>::read_ucd(std::istream &in,
                                numbers::internal_face_boundary_id));
 
           if (apply_all_indicators_to_manifolds)
```

Comment:
I read through the list of changes again and this is the only part I don't understand. Would you add a comment to help explain why this works?